react/react-training/src/redux/login/actions.js

```diff
@@ -1,55 +1,31 @@
 import UserService from '@services/LoginService.js';
-import { ERROR_READING_RESPONSE, MSG_UNKNOWN_ID } from '@screens/Login/validation';
+import { completeTypes, createTypes, withPostSuccess } from 'redux-recompose';
 
-export const actionTypes = {
-  // Login Service
-  USER_LOGIN: 'USER_LOGIN',
-  USER_LOGIN_FAILURE: 'USER_LOGIN_FAILURE',
-  USER_LOGIN_SUCCESS: 'USER_LOGIN_SUCCESS',
-
-  // Other actions
-  USER_LOGIN_VERIFY: 'USER_LOGIN_VERIFY',
-  USER_LOGOUT: 'USER_LOGOUT'
-};
+const myActionTypes = ['USER_LOGIN'];
```

Comment:
I think that isn't necessary this variable, you can assign directly in the `completeTypes`